ports/godot-cpp/portfile.cmake

```
@@ -0,0 +1,39 @@
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO "godotengine/godot-cpp"
+    REF "godot-${VERSION}-stable"
+    SHA512 "820e07ffb0545324f01598898bb342d7e143dcc8b83818824e7e1bc22937d3e8016b435f1ec085ebaae8b26e6f6dfb5500f120089316fc0f0c4153c340226941"
+    HEAD_REF "master"
+    PATCHES
+        "0001-Patches-so-that-godot-cpp-is-packagable.patch"
+)
+
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    INVERTED_FEATURES
+        "exceptions" GODOT_DISABLE_EXCEPTIONS
+)
+
+vcpkg_find_acquire_program("PYTHON3")
```

I need a little context here. Why is python needed for this port?

Generally speaking, `vcpkg_find_acquire_program(PYTHON3)` is used if:

1. you running a python script for the sake of running a python script
2. don't care about the version of Python you get
3. you aren't trying to build any modules into it
4. You aren't trying to link it into a resulting program
5. You aren't trying to read any results from it and assume they hold on the target
6. etc.

All other cases, this needs to depend on the python port.

---

This project uses python only for codegen.

---

```suggestion
vcpkg_find_acquire_program(PYTHON3)
```
This has relevance for ABI tracking, by literally matching filenames in `scripts/cmake`.